src/components/Legend/Legend.less

```diff
@@ -39,6 +39,7 @@
 	}
 
 	&-Item-indicator {
+		flex: 1 0 auto;
```

Comment:
Fixed another small bug where the indicator could get squished with long titles.